server/message_hooks.go

```diff
@@ -551,23 +550,7 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post) (
 }
 
 func (p *Plugin) handlePromptForConnection(userID, channelID string) {
-	promptInterval := p.getConfiguration().PromptIntervalForDMsAndGMs
-	if promptInterval <= 0 {
-		return
-	}
-
-	timestamp, err := p.store.GetDMAndGMChannelPromptTime(channelID, userID)
-	if err != nil {
-		p.API.LogWarn("Unable to get the last prompt timestamp for the channel", "channel_id", channelID, "error", err.Error())
-	}
-
-	if time.Until(timestamp) < -time.Hour*time.Duration(promptInterval) {
-		p.sendBotEphemeralPost(userID, channelID, "Your Mattermost account is not connected to MS Teams so your activity will not be relayed to users on MS Teams. You can connect your account using the `/msteams connect` slash command.")
-
-		if err = p.store.StoreDMAndGMChannelPromptTime(channelID, userID, time.Now()); err != nil {
-			p.API.LogWarn("Unable to store the last prompt timestamp for the channel", "channel_id", channelID, "error", err.Error())
-		}
-	}
+	p.sendBotEphemeralPost(userID, channelID, "Some users in this conversation rely on Microsoft Teams to receive your messages, but your account is not connected. Connect your account using the `/msteams connect` slash command.")
```

Comment:
@cwarnermm, might I again get your feedback on the proposed language here?

Reply:
@lieut-data - For this one, I propose: "Some users in this conversation rely on Microsoft Teams to receive your messages, but your account isn't connected. Connect your account using the `/msteams connect` slash command."